### DISCUSSION TITLE
Table detection problem via wordpress plugin

### DISCUSSION THREAD

**Discussion by totololynx** [stuttter/ludicrousdb#166] [General]
Hello, I would like to know why wordpress does not detect other databases via plugins. I use this config but wordpress does not detect tables from other databases. Thanks in advance. Cordially.

**Comment by JJJ** [comment]
LudicrousDB does not make WordPress or a plugin enumerate every table in every configured physical database. It routes a query to a logical dataset after WordPress or the plugin has decided which table it wants.

That distinction matters for plugins that inspect `$wpdb->tables()`, query `information_schema`, or run `SHOW TABLES`: adding a database to LudicrousDB does not add arbitrary plugin tables to WordPress's built-in table lists.

If this is still an issue, could you share the plugin name, the exact table it is trying to detect, what “does not detect” looks like, and a sanitized `db-config.php` showing the dataset and callback? Please remove hosts, usernames, passwords, and physical database names. With those details we can tell whether the plugin's discovery query needs routing or whether it is relying on a core table list that LudicrousDB should not change.
